### DISCUSSION TITLE
EI System Initialization

### DISCUSSION THREAD

**Discussion by TimThacker** [CURENT/andes#407] [Modeling]
Hantao,

Upon attempting to simulate the provided case for the eastern interconnection, I received the warning "Suspect initialization issue!" and the plotted results were not accurate. Is the data provided correct or has it been verified? Thanks.

**Comment by cuihantao** [comment]
Thanks for reporting. You can trace the issue by analyzing the output:

```
Suspect initialization issue! Simulation may crash!

     Name      | Var. Value | Eqn. Mismatch
---------------+------------+--------------
 pout IEEEG1 9 | 379.283    | -112.182     
```

It has to do with IEEEG1 9's pout variable not satisfying the equation. These are typically due to limits that are too low. You can check IEEEG1 and adjust the limits accordingly. Can you chek it and make a PR? These models are not used very often though. I'm glad that it might be useful to you.

**Comment by jinningwang** [comment]
Hi Tim,

As a further comment for Hantao's response, there is another TDS initialization error case at [Three-phase fault at desired line position, some strange TDS results #386](https://github.com/cuihantao/andes/discussions/386).
As a summary, I also have a blog discussing such issues at [Fix Power Flow Divergence and TDS Initialization Error in ANDES](https://jinningwang.github.io/posts/2022/11/andes_init_debug/).
Hope these materials help you.

Regards,
Jinning
